org-ql-completing-read.el

```diff
@@ -176,6 +176,7 @@ single predicate)."
                               (setf input (concat query-prefix input)))
                             (pcase flag
                               ('metadata (list 'metadata
+                                               (cons 'category 'org-heading)
```

Comment:
I love how Embark integration boils down to only a single line here :partying_face:

Reply:
Me too. :)